pubtools/_pulp/tasks/push/items/productid.py

```diff
@@ -80,8 +79,7 @@ class PulpProductIdPushItem(PulpDirectUploadPushItem):
     @product_versions.default
     def _product_versions_from_cert(self):
         if self.pushsource_item and self.pushsource_item.src:
```

Comment:
I think this shouldn't check `src` any more.  Now that there is a proper attribute for `products`, it should just be trusted and we shouldn't look at other attributes on the item to decide whether or not to use the `products` attribute.